docs/guide/running-extern-robot-controllers.md

```diff
@@ -0,0 +1,54 @@
+# Running Extern Robot Controllers
+
+This chapter describes extern robot controllers and how to use them.
+
+## Introduction
+
+Normally, Webots launches automatically the robot controller specified in the `controller` field of each [Robot](../reference/robot.md) node.
+However, if this field is set to `<extern>`, no controller is launched and the robot will behave like if its `controller` field was an empty string, that is, the robot will not be controlled.
```

Comment:
Do we really want the simulation to run while 'waiting' for the controller?
This doesn't seem really reproducible (which is a key element, especially when debugging).

Reply:
Yes, I believe this is the best way to go, because it opens all the possibilities to the user:
1. If you want to have a reproducible setup, you simply have to start the simulation in pause mode, connect the controller and run it.
2. If you want to have a world into which controllers can connect any time, while the simulation is running, this is also possible.

Reply:
Thank you for the explanation, that's fine for me. But in that case in the debugging documentation, we should say to the user that it is recommended to pause the simulation (because now we explicitly say the contrary).